core/src/Network/GRPC/LowLevel/Call.hs

```diff
@@ -181,7 +181,7 @@ mgdPtr = managed (bracket malloc free)
 
 serverCallIsExpired :: ServerCall a -> IO Bool
 serverCallIsExpired sc = do
-  currTime <- getTime Monotonic
+  C.CTimeSpec currTime <- bracket (C.gprNow C.GprClockMonotonic) C.timespecDestroy peek
```

Comment:
This isn’t strictly necessary but I changed it while debugging some tests and this seems like a better option than relying on `clock` and `gRPC` to agree on the clock.

Reply:
This seems like a reasonable change to make.